fog/view/server/src/db_fetcher.rs

```diff
@@ -282,64 +291,69 @@ impl<DB: RecoveryDb + Clone + Send + Sync + 'static> DbFetcherThread<DB> {
             // Attempt to load data for the next block.
             let get_tx_outs_by_block_result = {
                 let _metrics_timer = counters::GET_TX_OUTS_BY_BLOCK_TIME.start_timer();
-                self.db
-                    .get_tx_outs_by_block_and_key(ingress_key, block_index)
+                self.db.get_tx_outs_by_block_range_and_key(
+                    ingress_key,
+                    block_index,
+                    self.block_query_batch_size,
+                )
             };
 
             match get_tx_outs_by_block_result {
-                Ok(Some(tx_outs)) => {
-                    let num_tx_outs = tx_outs.len();
+                Ok(block_results) => {
+                    if block_results.is_empty() {
+                        continue;
+                    };
```

Comment:
Nit: Unnecessary braces
```suggestion
                    if block_results.is_empty() continue;
```

Reply:
cargo fmt does not agree with you, sorry